.dagger/main.go

```diff
@@ -43,12 +43,19 @@ func (m *ContainerUse) Release(ctx context.Context,
 	// GitHub org name for package publishing, set only if testing release process on a personal fork
 	//+default="dagger"
 	githubOrgName string,
+	// Chocolatey API key for Windows package publishing
```

Comment:
🔧 **Maintainability (Low):** Comment was correctly updated to match the function name, but this appears to be an unrelated fix bundled in this PR.

```suggestion
// Lint runs the linter
```